vnext/ReactUWP/Views/ViewViewManager.cpp

```diff
@@ -175,29 +175,22 @@ class ViewShadowNode : public ShadowNodeBase {
         winrt::make<winrt::react::uwp::implementation::ViewControl>();
 
     m_contentControlGotFocusRevoker =
-        contentControl.GotFocus(winrt::auto_revoke, [=](auto &&, auto &&) {
-          DispatchEvent(
-              "topFocus", std::move(folly::dynamic::object("target", m_tag)));
+        contentControl.GotFocus(winrt::auto_revoke, [=](auto &&, auto &&args) {
+          if (args.OriginalSource().try_as<winrt::UIElement>() ==
+              contentControl.as<winrt::UIElement>()) {
+            auto tag = m_tag;
+            DispatchEvent(
+                "topFocus", std::move(folly::dynamic::object("target", tag)));
+          }
         });
 
     m_contentControlLostFocusRevoker =
-        contentControl.LostFocus(winrt::auto_revoke, [=](auto &&, auto &&) {
-          DispatchEvent(
-              "topBlur", std::move(folly::dynamic::object("target", m_tag)));
-        });
-
-    // FUTURE: The space/enter handler can be removed in August 2019, still here
-    // to enable transitioning to new events if native code is updated before JS
-    m_contentControlKeyDownRevoker = contentControl.KeyDown(
```

Comment:
I'm probably mis-reading. does removing this mean that View's  will never emit topClick events? Does this impact using TouchableOpacity and the onPress event?

Reply:
A month or so ago we added code in the Js side to listen to key events in touchable*.uwp.js.  Whats actually happening right now with keyboard input is the Onpress event fires twice, once from this and again from the js integration... removing this will get us to only firing once